tasks/steps/copy2aegisub.py

```diff
@@ -22,23 +22,41 @@ def run_step(context: dict):
         appdirs.user_data_dir("Aegisub", "", roaming=True), "automation"
     )
     pyegi_dependency_path = os.path.join(automation_path, "dependency", "Pyegi")
+    pyegi_path = os.path.join(pyegi_dependency_path, "Pyegi")
 
     # Remove previous files
     shutil.rmtree(os.path.join(automation_path, "include", "Pyegi"), ignore_errors=True)
-    shutil.rmtree(os.path.join(pyegi_dependency_path, "Pyegi"), ignore_errors=True)
+    shutil.rmtree(pyegi_path, ignore_errors=True)
```

Comment:
I think this will cause problems if some scripts are already in that directory. Might be better to automatically make a list of what files we have in our release and for each existing directory on the user's machine, check if any of the files or sub-directories are exist in our release. If not, don't remove that directory. If yes, delete the existing files and keep going into sub-directories. Or whatever else that would work better. We could also consider the part in `pyproject.toml` files, which refers to keeping files or not right now instead of postponing it to later releases if it's not too much work.

Reply:
این تسک صرفا برای دلوپمنته. جدا از اون اصلا اسکریپتا رو هم پاک نمی‌کنه...